pkgs/tools/archivers/afio/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl } :
+
+stdenv.mkDerivation rec {
+  version = "2.5.1";
+  name = "afio";
```

Comment:
The version needs to be a part of `name`, as in `name = "afio-${version}"`; the attribute itself currently holds no special meaning.